addons/calendar/wizard/mail_activity_schedule_wizard.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo.tools.translate import _
+
+
+class MailActivityScheduleWizard(models.TransientModel):
+    _inherit = 'mail.activity.schedule.wizard'
+
+    calendar_event_id = fields.Many2one(related='activity_id.calendar_event_id')
+
+    def action_create_calendar_event(self):
+        self.ensure_one()
+        if self.is_batch_mode:
+            raise ValueError("Not allowed in in batch mode.")
```

Comment:
Beware translations :) would also make an UserError as I suppose people could encounter it
```suggestion
            raise ValueError(_("Not allowed in in batch mode."))
```